src/GitReportCL/GitDiffOutputBuilder.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.IO;
+namespace GitReport.CLI
+{
+    class JsonConfig
+    {
+        public static IConfiguration Configuration { get; set; }
+        Dictionary<string, string> ids = new Dictionary<string, string>();
+        public JsonConfig()
+        {
+            var builder = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory())
+            .AddJsonFile("jsconfig.json", optional: true, reloadOnChange: true);
+            Configuration = builder.Build();
+
+            CreateIdsDictionary();
+        }
+        public bool TryMatchPaths(string pathFromProcess, out string finalId)
```

Comment:
It should have singluar for `TryMatchPath` because you are giving on path not multple of them